services/src/main/scala/cromwell/services/metadata/impl/MetadataServiceActor.scala

```diff
@@ -161,6 +177,16 @@ case class MetadataServiceActor(serviceConfig: Config, globalConfig: Config, ser
     }
   }
 
+  private def sendMetadataTableSizeMetrics(): Unit = {
+    def convertToGiB(bytes: Long): Long = bytes / (1024 * 1024 * 1024)
+
+    getMetadataTableSizeInformation foreach { v =>
```

Comment:
I think right now this would be throwing away any error information if something goes wrong in the lookup. Is it possible to at least `warn` log about it?